pages.id/common/git-branch.md

```diff
@@ -1,17 +1,17 @@
 # git branch
 
-> Perintah Git utama untuk bekerja dengan cabang (*branch*).
+> Perintah Git utama untuk bekerja dengan cabang (_branch_).
 > Informasi lebih lanjut: <https://git-scm.com/docs/git-branch>.
 
-- Menampilkan daftar cabang lokal. Cabang saat ini ditandai oleh `*`:
+- Tampil daftar semua cabang (local dan remote; cabang saat ini ditandai oleh `*`) :
 
-`git branch`
+`git branch --all`
 
-- Menampilkan daftar semua cabang (lokal dan remote):
+- Tampil daftar semua cabang yang memiliki komit tertentu di dalam riwayat:
```

Comment:
```suggestion
- Tampil daftar semua cabang yang memiliki Git komit tertentu di dalam riwayat:
```
Feel free to change the work placement of `Git`.

> Note: I used Google translate to check the translations and I am not a native speaker.

Reply:
```suggestion
- Tampil daftar semua cabang yang memiliki komit Git tertentu di dalam riwayat:
```

Shouldn't this be "komit Git" instead of "Git komit"?

Reply:
> Shouldn't this be "komit Git" instead of "Git komit"?

Thanks for the suggestion, I will update my comment.

Reply:
addressed! Thank you!